# UCS Virtual Machine Manager Daemon.
#
# Copyright 2010-2024 Univention GmbH
#
# https://www.univention.de/
#
# All rights reserved.
#
# The source code of the software contained in this package
# as well as the source package itself are made available
# under the terms of the GNU Affero General Public License version 3
# (GNU AGPL V3) as published by the Free Software Foundation.
#
# Binary versions of this package provided by Univention to you as
# well as other copyrighted, protected or trademarked materials like
# Logos, graphics, fonts, specific documentations and configurations,
# cryptographic keys etc. are subject to a license agreement between
# you and Univention.
#
# In the case you use the software under the terms of the GNU AGPL V3,
# the program is provided in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public
# License with the Debian GNU/Linux or Univention distribution in file
# /usr/share/common-licenses/AGPL-3; if not, see
# <https://www.gnu.org/licenses/>.
#
#
# Philipp Hahn <hahn@univention.de>, 2010.
# Nicolas Ternisien <nicolas@bureau-cornavin.com>, 2016.
# Éric Bischoff <ebischoff@bureau-cornavin.com>, 2016.
# Nicolas Ternisien <nicolas@workshop-chapina.com>, 2017.
msgid ""
msgstr ""
"Project-Id-Version: univention-virtual-machine-manager 0.9.333-1\n"
"Report-Msgid-Bugs-To: forge.univention.org\n"
"POT-Creation-Date: 2017-04-14 00:29+0200\n"
"PO-Revision-Date: 2017-05-23 22:24+0100\n"
"Last-Translator: Nicolas Ternisien <nicolas@workshop-chapina.com>\n"
"Language-Team: French <fr@workshop-chapina.com>\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Lokalize 2.0\n"

#: src/univention-virtual-machine-manager-daemon:212
msgid "Cache options"
msgstr "Options du cache"

#: src/univention/uvmm/commands.py:630
#, python-format
msgid "Command \"%(command)s\" is restricted"
msgstr "L'utilisation de la commande « %(command)s » est restreinte"

#: src/univention-virtual-machine-manager-daemon:206
msgid "Connection options"
msgstr "Options de la connexion"

#: src/univention/uvmm/uvmm_ldap.py:194 src/univention/uvmm/uvmm_ldap.py:250
#: src/univention/uvmm/uvmm_ldap.py:284 src/univention/uvmm/uvmm_ldap.py:301
msgid "Could not open LDAP-Admin connection"
msgstr "Impossible d'ouvrir une connexion LDAP-Admin"

#: src/univention/uvmm/uvmm_ldap.py:175
msgid "Could not open LDAP-Machine connection"
msgstr "Impossible d'ouvrir une connexion LDAP-Machine"

#: src/univention/uvmm/client.py:124
#, python-format
msgid "Could not open socket \"%(path)s\": %(errno)d"
msgstr "Impossible d'ouvrir la socket « %(path)s » : %(errno)d"

#: src/univention/uvmm/uvmm_ldap.py:166
#, python-format
msgid "Could not query \"%(uri)s\""
msgstr "Impossible d'envoyer une requête à « %(uri)s »"

#: src/univention/uvmm/client.py:66
#, python-format
msgid "Could not send request: %(errno)d"
msgstr "Impossible d'envoyer la requête : %(errno)d"

#: src/univention-virtual-machine-manager-daemon:221
msgid "Daemon options"
msgstr "Options du démon"

#: src/univention-virtual-machine-manager-daemon:218
msgid "Do not load cached state"
msgstr "Ne pas charger l'état en cache"

#: src/univention/uvmm/node.py:1710
#, python-format
msgid "Domain \"%(domain)s\" already exists: %(uuid)s"
msgstr "Le domaine « %(domain)s » existe déjà : %(uuid)s"

#: src/univention/uvmm/node.py:1554
#, python-format
msgid "Domain \"%(domain)s\" in state \"%(state)s\" can not be migrated"
msgstr "Le domaine « %(domain)s » dans l'état « %(state)s » ne peut être migré"

#: src/univention/uvmm/node.py:1718
#, python-format
msgid "Domain \"%(domain)s\" is not shut off: %(state)d"
msgstr "Le domaine « %(domain)s » n'est pas éteint : %(state)d"

#: src/univention/uvmm/node.py:1222
#, python-format
msgid "Domain name \"%(domain)s\" already used by \"%(uuid)s\""
msgstr "Le nom de domaine « %(domain)s » est déjà utilisé par « %(uuid)s »"

#: src/univention/uvmm/client.py:97
msgid "EOS while waiting for answer."
msgstr "Signal « Fin de la connexion » reçu en attendant la réponse."

#: src/univention/uvmm/node.py:1892
#, python-format
msgid "Error cloning \"%(domain)s\": %(error)s"
msgstr "Erreur en dupliquant « %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1595
#, python-format
msgid "Error creating \"%(domain)s\" snapshot: %(error)s"
msgstr "Erreur en créant un instantané de « %(domain)s » : %(error)s"

#: src/univention/uvmm/storage.py:95
#, python-format
msgid "Error creating storage pool \"%(pool)s\": %(error)s"
msgstr ""
"Erreur en créant l'espace de stockage mis en commun « %(pool)s » : %(error)s"

#: src/univention/uvmm/storage.py:275
#, python-format
msgid ""
"Error creating storage volume \"%(name)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en créant le volume de stockage « %(name)s » de « %(domain)s » : "
"%(error)s"

#: src/univention/uvmm/node.py:1312
#, python-format
msgid "Error defining domain \"%(domain)s\": %(error)s"
msgstr "Erreur en définissant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1646 src/univention/uvmm/node.py:1655
#, python-format
msgid "Error deleting \"%(domain)s\" snapshot: %(error)s"
msgstr "Erreur en supprimant l'instantané « %(domain)s » : %(error)s"

#: src/univention/uvmm/storage.py:425 src/univention/uvmm/storage.py:448
#: src/univention/uvmm/storage.py:469
#, python-format
msgid "Error listing pools at \"%(uri)s\": %(error)s"
msgstr ""
"Erreur en obtenant la liste des espaces mis en commun de « %(uri)s » : "
"%(error)s"

#: src/univention/uvmm/storage.py:289 src/univention/uvmm/storage.py:309
#, python-format
msgid "Error listing volumes at \"%(uri)s\": %(error)s"
msgstr "Erreur en obtenant la liste des volumes de « %(uri)s » : %(error)s"

#: src/univention/uvmm/storage.py:147
#, python-format
msgid "Error locating storage pool \"%(pool)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en localisant l'espace de stockage mis en commun « %(pool)s » pour "
"« %(domain)s » : %(error)s"

#: src/univention/uvmm/storage.py:123 src/univention/uvmm/storage.py:185
#, python-format
msgid ""
"Error locating storage volume \"%(volume)s\" for \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en localisant le volume de stockage « %(volume)s » pour "
"« %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1449
#, python-format
msgid "Error managing domain \"%(domain)s\""
msgstr "Erreur en gérant le domaine « %(domain)s »"

#: src/univention/uvmm/node.py:1452 src/univention/uvmm/node.py:1455
#, python-format
msgid "Error managing domain \"%(domain)s\": %(error)s"
msgstr "Erreur en gérant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1543 src/univention/uvmm/node.py:1573
#, python-format
msgid "Error migrating domain \"%(domain)s\": %(error)s"
msgstr "Erreur en migrant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/uvmm_ldap.py:130
#, python-format
msgid "Error reading %(file)s: %(msg)s"
msgstr "Erreur en lisant %(file)s : %(msg)s"

#: src/univention/uvmm/uvmm_ldap.py:133
#, python-format
msgid "Error reading incomplete %(file)s."
msgstr "Erreur en lisant le fichier incomplet %(file)s."

#: src/univention/uvmm/uvmm_ldap.py:105
#, python-format
msgid "Error removing %(file)s.old: %(msg)s"
msgstr "Erreur en supprimant %(file)s.old : %(msg)s"

#: src/univention/uvmm/node.py:1303
#, python-format
msgid "Error removing domain \"%(domain)s\": %(error)s"
msgstr "Erreur en supprimant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/uvmm_ldap.py:115
#, python-format
msgid "Error renaming %(file)s.new: %(msg)s"
msgstr "Erreur en renommant %(file)s.new : %(msg)s"

#: src/univention/uvmm/uvmm_ldap.py:110
#, python-format
msgid "Error renaming %(file)s: %(msg)s"
msgstr "Erreur en renommant %(file)s : %(msg)s"

#: src/univention/uvmm/node.py:1485
#, python-format
msgid "Error restoring domain \"%(domain)s\": %(error)s"
msgstr "Erreur en restaurant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/network.py:56
#, python-format
msgid "Error retrieving list of active networks: %(error)s"
msgstr "Erreur en obtenant la liste des réseaux actifs : %(error)s"

#: src/univention/uvmm/network.py:79
#, python-format
msgid "Error retrieving list of networks: %(error)s"
msgstr "Erreur en obtenant la liste des réseaux : %(error)s"

#: src/univention/uvmm/node.py:1227 src/univention/uvmm/node.py:1236
#, python-format
msgid "Error retrieving old domain \"%(domain)s\": %(error)s"
msgstr "Erreur en obtenant l'ancien domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1622 src/univention/uvmm/node.py:1628
#, python-format
msgid "Error reverting \"%(domain)s\" to snapshot: %(error)s"
msgstr "Erreur en ramenant « %(domain)s » vers l'instantané : %(error)s"

#: src/univention/uvmm/node.py:1470
#, python-format
msgid "Error saving domain \"%(domain)s\": %(error)s"
msgstr "Erreur en enregistrant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/network.py:71
#, python-format
msgid "Error starting network %(name)s: %(error)s"
msgstr "Erreur en démarrant le réseau %(name)s : %(error)s"

#: src/univention/uvmm/node.py:1514
#, python-format
msgid "Error undefining domain \"%(domain)s\": %(error)s"
msgstr ""
"Erreur en supprimant la définition du domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/node.py:1674 src/univention/uvmm/node.py:1691
#, python-format
msgid "Error updating domain \"%(domain)s\""
msgstr "Erreur en actualisant le domaine « %(domain)s »"

#: src/univention/uvmm/node.py:1278 src/univention/uvmm/node.py:1281
#: src/univention/uvmm/node.py:1286
#, python-format
msgid "Error updating domain \"%(domain)s\": %(error)s"
msgstr "Erreur en actualisant le domaine « %(domain)s » : %(error)s"

#: src/univention/uvmm/client.py:107
#, python-format
msgid "Error while closing socket: %(errno)d"
msgstr "Erreur en fermant la socket : %(errno)d"

#: src/univention/uvmm/client.py:95
#, python-format
msgid "Error while waiting for answer: %(errno)d"
msgstr "Erreur en attendant la réponse : %(errno)d"

#: src/univention/uvmm/unix.py:154
#, python-format
msgid "Exception: %(exception)s"
msgstr "Exception : %(exception)s"

#: src/univention/uvmm/node.py:909
#, python-format
msgid "Failed to backup domain %(domain)s: %(xml)s"
msgstr "Échec de la sauvegarde du domaine %(domain)s : %(xml)s"

#: src/univention/uvmm/node.py:1865
#, python-format
msgid "Failed to clone volume \"%(volume)s\": %(error)s"
msgstr "Échec de la duplication du volume « %(volume)s » : %(error)s"

#: src/univention-virtual-machine-manager-daemon:124
msgid "Failed to fork daemon."
msgstr "Impossible de scinder le démon."

#: src/univention/uvmm/node.py:1830
#, python-format
msgid "Failed to generate new name for disk \"%(disk)s\""
msgstr "Échec de génération d'un nouveau nom pour le disque « %(disk)s »"

#: src/univention/uvmm/node.py:1272
msgid "Failed to update device."
msgstr "Échec de mise à jour du périphérique."

#: src/univention/uvmm/node.py:1696
#, python-format
msgid "Failed to update domain \"%(domain)s\""
msgstr "Échec d'actualisation du domaine « %(domain)s »"

#: src/univention/uvmm/node.py:1331 src/univention/uvmm/node.py:1334
msgid "Failed to update the additionally information in the LDAP directory."
msgstr ""
"Échec d'actualisation des informations supplémentaires dans l'annuaire LDAP."

#: src/univention-virtual-machine-manager-daemon:227
msgid "Fork into background"
msgstr "Scinder en arrière-plan"

#: src/univention/uvmm/node.py:841
#, python-format
msgid "Hypervisor \"%(uri)s\" is already connected."
msgstr "L'hyperviseur « %(uri)s » est déjà connecté."

#: src/univention/uvmm/node.py:853 src/univention/uvmm/node.py:862
#, python-format
msgid "Hypervisor \"%(uri)s\" is not connected."
msgstr "L'hyperviseur « %(uri)s » n'est pas connecté."

#: src/univention/uvmm/protocol.py:94
#, python-format
msgid "Incompatible version: %(major)d.%(minor)d"
msgstr "Version non compatible : %(major)d.%(minor)d"

#: src/univention/uvmm/uvmm_ldap.py:141
msgid "No SERVICES defined."
msgstr "Aucun SERVICE défini."

#: src/univention-virtual-machine-manager:108
msgid "No nodes found."
msgstr "Aucun nœud trouvé."

#: src/univention/uvmm/protocol.py:105
#, python-format
msgid "Not a Packet: %(type)s"
msgstr "N'est pas un paquet : %(type)s"

#: src/univention/uvmm/client.py:87
msgid "Not a UVMM_Response."
msgstr "N'est pas une réponse UVMM."

#: src/univention/uvmm/protocol.py:103
#, python-format
msgid "Not a valid Packet: %(msg)s"
msgstr "N'est pas un paquet valide : %(msg)s"

#: src/univention/uvmm/unix.py:107
#, python-format
msgid "Packet is no UVMM Request: %(type)s"
msgstr "Le paquet n'est pas une requête UVMM : %(type)s"

#: src/univention-virtual-machine-manager:136
msgid "Path to the UNIX socket"
msgstr "Emplacement de la socket UNIX"

#: src/univention-virtual-machine-manager-daemon:209
#, python-format
msgid "Path to the UNIX socket [%(default)s]"
msgstr "Emplacement de la socket UNIX [%(default)s]"

#: src/univention-virtual-machine-manager-daemon:236
#, python-format
msgid "Path to the ini-file [%(default)s]"
msgstr "Emplacement du fichier « ini » [%(default)s]"

#: src/univention-virtual-machine-manager-daemon:230
msgid "Path to the log-file"
msgstr "Emplacement du fichier de journalisation"

#: src/univention-virtual-machine-manager-daemon:224
#, python-format
msgid "Path to the pid-file [%(default)s]"
msgstr "Emplacement du fichier de PID [%(default)s]"

#: src/univention-virtual-machine-manager:142
#: src/univention-virtual-machine-manager-daemon:233
msgid "Print additional information"
msgstr "Afficher des informations supplémentaires"

#: src/univention-virtual-machine-manager-daemon:239
msgid "Skip reading ini-file"
msgstr "Ignorer la lecture du fichier « ini »"

#: src/univention/uvmm/node.py:1581 src/univention/uvmm/node.py:1586
#: src/univention/uvmm/node.py:1603 src/univention/uvmm/node.py:1608
#: src/univention/uvmm/node.py:1636 src/univention/uvmm/node.py:1641
#, python-format
msgid "Snapshot not supported \"%(node)s\""
msgstr "L'instantané n'est pas pris en charge « %(node)s »"

#: src/univention/uvmm/ec2cloud.py:507
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"AuthFailure: The provided AWS access credentials could not be validated. "
"Please ensure that you are using the correct access keys. Consult the AWS "
"service documentation for details."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"AuthFailure : les preuves d'identité pour l'accès à AWS fournies n'ont pu "
"être validées. Veuillez vous assurer que vous utilisez des clés d'accès "
"correctes. Consultez la documentation du service AWS pour plus de détails."

#: src/univention/uvmm/ec2cloud.py:540
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"RequestExpired: Please check your system time to interact with AWS."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"RequestExpired: veuillez vérifier l'horloge de votre système pour interagir "
"avec AWS."

#: src/univention/uvmm/ec2cloud.py:542
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"UnauthorizedOperation: The provided AWS access credentials are not "
"authorized to perform this operation. Check your IAM policies, and ensure "
"that you are using the correct access keys. Also, the IAM user must have "
"appropriate access rights to interact with EC2, e.g. AmazonEC2FullAccess."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"UnauthorizedOperation : les preuves d'identité pour l'accès à AWS fournies "
"ne permettent pas d'effectuer cette opération. Vérifiez vos droits IAM et "
"assurez-vous que vous utilisez des clés d'accès correctes. De plus, "
"l'utilisateur IAM doit avoir des droits d'accès appropriés pour interagir "
"avec EC2, par ex. AmazonEC2FullAccess."

#: src/univention/uvmm/ec2cloud.py:538
#, python-format
msgid ""
"The EC2 region returned an error for connection \"%(connection)s\":\n"
"\n"
"Your AWS account is currently blocked. If you have questions, please contact "
"AWS Support."
msgstr ""
"La région EC2 a retourné une erreur pour la connexion « %(connection)s » :\n"
"\n"
"Votre compte AWS est actuellement bloqué. Si vous avez des questions, "
"veuillez contacter le service en ligne d'AWS."

#: src/univention/uvmm/client.py:105
msgid "Timed out while closing socket."
msgstr "Délai dépassé en fermant la socket."

#: src/univention/uvmm/client.py:122
#, python-format
msgid "Timed out while opening local socket \"%(path)s\"."
msgstr "Délai dépassé en ouvrant une socket locale « %(path)s »."

#: src/univention/uvmm/client.py:93
msgid "Timed out while receiving data."
msgstr "Délai dépassé en recevant les données."

#: src/univention/uvmm/client.py:64
msgid "Timed out while sending data."
msgstr "Délai dépassé en envoyant les données."

#: src/univention-virtual-machine-manager:149
msgid "Timeout in seconds for UVMM commands"
msgstr "Délai à ne pas dépasser en secondes pour les commandes UVMM"

#: src/univention/uvmm/commands.py:634
#, python-format
msgid "Unknown command \"%(command)s\""
msgstr "Commande « %(command)s » inconnue"

#: src/univention/uvmm/node.py:373
#, python-format
msgid "Unknown disk type: %(type)d"
msgstr "Type de disque inconnu : %(type)d"

#: src/univention/uvmm/node.py:1094 src/univention/uvmm/node.py:1800
#, python-format
msgid "Unknown disk/type='%(type)s'"
msgstr "Type de disque inconnu='%(type)s'"

#: src/univention/uvmm/node.py:1365
#, python-format
msgid "Unknown domain \"%s\""
msgstr "Domaine inconnu « %s »"

#: src/univention/uvmm/node.py:1139
#, python-format
msgid "Unknown interface/type='%(type)s'"
msgstr "Type d'interface inconnu='%(type)s'"

#: src/univention/uvmm/node.py:1007
#, python-format
msgid "Unknown os/type='%(type)s'"
msgstr "Type de système d'exploitation inconnu='%(type)s'"

#: src/univention/uvmm/node.py:1412
#, python-format
msgid "Unsupported state transition %(cur_state)s to %(next_state)s"
msgstr ""
"Transition de l'état %(cur_state)s vers %(next_state)s non prise en charge"

#: src/univention/uvmm/storage.py:249
#, python-format
msgid "Unsupported storage-pool-type \"%(pool_type)s\" for \"%(domain)s\""
msgstr ""
"storage-pool-type « %(pool_type)s » non pris en charge pour « %(domain)s »"

#: src/univention/uvmm/node.py:1809
#, python-format
msgid "Volume \"%(volume)s\" not found: %(error)s"
msgstr "Volume « %(volume)s » non trouvé : %(error)s"

#: src/univention/uvmm/commands.py:64 src/univention/uvmm/commands.py:257
#, python-format
msgid "args != dict: %(args)s"
msgstr "args != dictionnaire : %(args)s"

#: src/univention-virtual-machine-manager-daemon:215
#, python-format
msgid "cache directory [%(default)s]"
msgstr "Dossier de cache [%(default)s]"

#: src/univention/uvmm/commands.py:105 src/univention/uvmm/commands.py:135
#: src/univention/uvmm/commands.py:149 src/univention/uvmm/commands.py:162
#: src/univention/uvmm/commands.py:175 src/univention/uvmm/commands.py:188
#: src/univention/uvmm/commands.py:201 src/univention/uvmm/commands.py:214
#: src/univention/uvmm/commands.py:227 src/univention/uvmm/commands.py:242
#: src/univention/uvmm/commands.py:255
#, python-format
msgid "conn_name != string: %(conn_name)s"
msgstr "conn_name != chaîne : %(conn_name)s"

#: src/univention/uvmm/commands.py:395
#, python-format
msgid "definition != Domain: %(domain)s"
msgstr "definition != domaine : %(domain)s"

#: src/univention/uvmm/commands.py:378 src/univention/uvmm/commands.py:412
#: src/univention/uvmm/commands.py:427 src/univention/uvmm/commands.py:442
#: src/univention/uvmm/commands.py:457 src/univention/uvmm/commands.py:478
#: src/univention/uvmm/commands.py:493 src/univention/uvmm/commands.py:508
#: src/univention/uvmm/commands.py:523 src/univention/uvmm/commands.py:536
#: src/univention/uvmm/commands.py:549
#, python-format
msgid "domain != string: %(domain)s"
msgstr "domain != chaîne : %(domain)s"

#: src/univention/uvmm/commands.py:121
#, python-format
msgid "freq != int: %(freq)s"
msgstr "freq != entier : %(freq)s"

#: src/univention/uvmm/commands.py:327
#, python-format
msgid "group != string: %(group)s"
msgstr "group != chaîne : %(group)s"

#: src/univention/uvmm/commands.py:314
#, python-format
msgid "hz != int: %(hz)s"
msgstr "hz != entier : %(hz)s"

#: src/univention/uvmm/commands.py:229 src/univention/uvmm/commands.py:244
#, python-format
msgid "instance_id != string: %(instance_id)s"
msgstr "instance_id != chaîne : %(instance_id)s"

#: src/univention/uvmm/commands.py:79 src/univention/uvmm/commands.py:123
#: src/univention/uvmm/commands.py:551
#, python-format
msgid "name != string: %(name)s"
msgstr "name != chaîne : %(name)s"

#: src/univention/uvmm/commands.py:92 src/univention/uvmm/commands.py:107
#: src/univention/uvmm/commands.py:329 src/univention/uvmm/commands.py:361
#, python-format
msgid "pattern != string: %(pattern)s"
msgstr "pattern != chaîne : %(pattern)s"

#: src/univention/uvmm/commands.py:495 src/univention/uvmm/commands.py:510
#: src/univention/uvmm/commands.py:525
#, python-format
msgid "snapshot != string: %(snapshot)s"
msgstr "snapshot != chaîne : %(snapshot)s"

#: src/univention/uvmm/commands.py:429 src/univention/uvmm/commands.py:444
#, python-format
msgid "statefile != string: %(file)s"
msgstr "statefile != chaîne : %(file)s"

#: src/univention/uvmm/commands.py:553
#, python-format
msgid "subst != dict: %(subst)s"
msgstr "subst != dictionnaire : %(subst)s"

#: src/univention/uvmm/commands.py:556 src/univention/uvmm/commands.py:558
#, python-format
msgid "subst[] != string: %(subst)s"
msgstr "subst[] != chaîne : %(subst)s"

#: src/univention/uvmm/commands.py:480
#, python-format
msgid "target_uri != string: %(uri)s"
msgstr "target_uri != chaîne : %(uri)s"

#: src/univention/uvmm/commands.py:67
#, python-format
msgid "testconnection is not a bool %(testconnection)s"
msgstr "testconnection n'est pas un booléen %(testconnection)s"

#: src/univention/uvmm/commands.py:297
#, python-format
msgid "unknown node %(uri)s"
msgstr "nœud inconnu %(uri)s"

#: src/univention/uvmm/commands.py:231 src/univention/uvmm/commands.py:414
#, python-format
msgid "unsupported state: %(state)s"
msgstr "état non pris en charge : %(state)s"

#: src/univention/uvmm/commands.py:267 src/univention/uvmm/commands.py:279
#: src/univention/uvmm/commands.py:291 src/univention/uvmm/commands.py:316
#: src/univention/uvmm/commands.py:359 src/univention/uvmm/commands.py:376
#: src/univention/uvmm/commands.py:393 src/univention/uvmm/commands.py:410
#: src/univention/uvmm/commands.py:425 src/univention/uvmm/commands.py:440
#: src/univention/uvmm/commands.py:455 src/univention/uvmm/commands.py:476
#: src/univention/uvmm/commands.py:491 src/univention/uvmm/commands.py:506
#: src/univention/uvmm/commands.py:521 src/univention/uvmm/commands.py:547
#: src/univention/uvmm/commands.py:573 src/univention/uvmm/commands.py:588
#: src/univention/uvmm/commands.py:605
#, python-format
msgid "uri != string: %(uri)s"
msgstr "uri != chaîne : %(uri)s"

#: src/univention-virtual-machine-manager-daemon:204
msgid "usage: %prog [options] [uri...]"
msgstr "utilisation : %prog [options] [uri...]"

#: src/univention/uvmm/commands.py:622
#, python-format
msgid "volume != string: %(volume)s"
msgstr "volume != chaîne : %(volume)s"

#: src/univention/uvmm/commands.py:459
#, python-format
msgid "volumes != list or None: %(volumes)s"
msgstr "volumes != liste ou None: %(volumes)s"

#: src/univention/uvmm/commands.py:463 src/univention/uvmm/commands.py:608
#, python-format
msgid "volumes[] != string: %(volume)s"
msgstr "volumes[] != chaîne : %(volume)s"

